cookbook/core/control_flow/subworkflows.py

```diff
@@ -1,55 +1,45 @@
 """
-Subworkflows
+Subworkflows 
 ------------
 
-Launch plans were introduced in the Basics section of this book. Subworkflows are similar in that they allow users
-to kick off one workflow from inside another. What's the difference? Think of launch plans as passing by pointer and
-subworkflows as passing by value.
+Subworkflows are similar to :ref:`launch plans <Launch plans>` in that they allow users to kick off one workflow from inside another. 
 
-.. note::
+What's the difference? 
+Think of launch plans as pass by pointer and subworkflows as pass by value.
 
-    the real reason why subworkflows exist is because this is exactly how dynamic workflows are handled by flyte. So
-    instead of hiding the functionality, we expose the functionality at the user level. There are pros and cons of
-    using subworkflows as described below
+.. note::
 
-When you include a launch plan of workflow A inside workflow B, when B gets run, a new workflow execution,
-replete with a new workflow execution ID, a new Flyte UI link, will be run.
```

Comment:
Have you removed this @SmritiSatyanV? I think this has to be copied to the "external workflow" section? It'd make sense to include it there, right?